packages/inter-protocol/test/vaultFactory/vaultFactoryUtils.js

```diff
@@ -108,7 +110,7 @@ export const setupElectorateReserveAndAuction = async (
     ClockStep: 2n,
     StartingRate: 10500n,
     LowestRate: 5500n,
-    DiscountStep: 2000n,
+    DiscountStep: discountStep || 2000n,
```

Comment:
the function signature said this is required. I think that's best but if you want to have a default param value please express express it in the signature.

Reply:
done